framework/include/ble_manager/ble_common.h

```diff
@@ -38,8 +38,17 @@
 #define BLE_ADV_RESP_DATA_MAX_LEN 31
 
 /* These are board specific values */
-#define BLE_MAX_CONNECTION_COUNT 3
-#define BLE_MAX_BONDED_DEVICE 10
+#ifdef CONFIG_BLE_CENTRAL_CONN_NUM
+#define BLE_MAX_CENTRAL_CONN_NUM CONFIG_BLE_CENTRAL_CONN_NUM
```

Comment:
I am sorry but This is VERY BAD approach, framework NEVER have dependency with H/W
Let's assume that we have a USB port and each of dongle has different number of connection.
If user changed dongle , what happened?????
So every init time of framework, you should ask h/w spec to driver.
In this time, you can use ioctl to get that info.

Reply:
Yes, you are right.
Hence 'CONFIG_BLE_CENTRAL_CONN_NUM' is set by realtek driver layer.
All options which has dependency with H/W are located in _os/drivers/bluetooth/realtek_ble/Kconfig_,
As following your example, if an user changes dongle, the driver code should be changed too including Kconfig.

Reply:
I mean various of h/w can be changed in real time, with one binary.
That's why most of platform has routine what checking h/w params when it initialized. F/W should not use config of h/w directly.

Reply:
I agree to Taejon's opinion. If new ble driver need to be ported then same configurations(ex. BLE_MAX_CENTRAL_CONN_NUM) should be defined to it too. So how about to define these configurations in BLE framework?